matrix/common/query_constructor.py

```diff
@@ -250,3 +250,7 @@ def filter_to_where(matrix_filter: typing.Dict[str, typing.Any]) -> str:
                 f"({filter_to_where(v)})" for v in value]) + ')'
     else:
         raise MalformedMatrixFilter(f"Invalid op: {op}")
+
+
+def list_to_query_str(values: typing.Iterable[str]) -> str:
```

Comment:
This will return an incorrect string if `values` is numeric. Do you think it's worth flagging or checking that?

Reply:
The type hint specifies `str` as input, but I agree it could be clearer. Renamed to `format_str_list` + added comment